src/index.scss

```diff
@@ -1,3 +1,6 @@
+@import "./movie-components/movie-container-component/_movie-container";
+@import "./movie-components/movie-card-component/_movie-card.scss";
```

Comment:
These imports make this file look clean! One small thing is that we have an extension on this import that's unnecessary.